editor/code_editor.cpp

```diff
@@ -1510,7 +1518,7 @@ void CodeTextEditor::toggle_inline_comment(const String &delimiter) {
 		// Comment/uncomment.
 		for (int line = from; line <= to; line++) {
 			String line_text = text_editor->get_line(line);
-			if (line_text.strip_edges().is_empty()) {
+			if (line_text.strip_edges().is_empty() && !is_commented) {
```

Comment:
Given this is no longer always adding / removing, do we need to change the caret readjustment below? 
`int offset = (is_commented ? -1 : 1) * delimiter.length();` particularly with multiple carets.

Reply:
Great Catch! I think it needs more investigation, I will take a look.

Reply:
I did some test and it seems fine because affected lines are empty lines so nothing weird happens. Could you help me to construct a scenario where my code fails ? I can take a look and decide how to update my code. Thanks !

Reply:
Sure I'm thinking of a situation like this:
```
1. |#some test
2. #code
3.    |      |
```
In this case there is one caret with a selection on lines 1-3. 
The second caret is on line 3.

When toggling comment in this situation, I would expect that the carets on line 3 will not change position.